test/cluster/test_tablets_colocation.py

```diff
@@ -419,6 +419,8 @@ async def test_repair_colocated_base_and_view(manager: ManagerClient):
 
         # Start node 2 back up
         await manager.server_start(servers[1].server_id)
+        await wait_for_cql_and_get_hosts(cql, servers, time.time() + 60)
```

Comment:
nit: I'm always confused why we don't have a simple flag for .server_start() that would cover both requests - wait till it's fully up (inc. all others seeing it).